### PR TITLE
{system} [dummy] rCUDA 4.0.1 & 5.0 (REVIEW)

### PR DESCRIPTION
**Pull request by fgeorgatos**
Mentioned by @georgets, during FZJ hackathon; had a template for it ready...

Signed-off-by: Fotis Georgatos <fotis.georgatos@uni.lu>

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/690f68dcf99fa69c3e6e for a full test report.

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/3903e7108989549ee34f for a full test report.

**Comment by fgeorgatos** [comment]
@boegel: this requires to obtain the tarball manually first, otherwise not unusual case.

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/r/rCUDA/rCUDA-4.0.1_linux_64_Ubuntu10.04.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+name = "rCUDA"
+version = "4.0.1"
+versionsuffix = "_linux_64_Ubuntu10.04"
+
+homepage = 'http://www.rcuda.net/'
+description = """The rCUDA Framework enables the concurrent usage of CUDA-compatible devices remotely."""
+
+toolchain = {'version': 'dummy', 'name': 'dummy'}
+
+sources = ['rCUDA_%(version)s%(versionsuffix)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/r/rCUDA/rCUDA-4.0.1_linux_64_Ubuntu10.04.eb]: include a comment with location where this can be downloaded from?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f6d26459aa1b42a5e919 for a full test report.

**Comment by boegel** [comment]
@fgeorgatos: the installation works fine, but there's a filename issue (`-` vs `_` after the software name), please fix and if Jenkins is happy I'll merge this in

```
AssertionError: Filename '.../rCUDA_5.0_linux_64_scientificLinux6.eb' of parsed easconfig matches expected filename 'rCUDA-5.0_linux_64_scientificLinux6.eb'
```

**Comment by fgeorgatos** [comment]
@boegel, indeed, thanks for notifying, done!

**Comment by boegel** [comment]
Going in, thanks @fgeorgatos!

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 2 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/b938673ca345831edcef for a full test report.

**Pull request merged**